### PR TITLE
Customize ticket background

### PR DESCRIPTION
**Pull request by shishanksingh2015**
Added two new attributes for changing background of ticket above and below the divider.
```app:ticketBackgroundAboveDivider```
```app:ticketBackgroundBelowDivider```
drawable, gradient or color can be added to ticket.
after_change
before_change

for issue https://github.com/vipulasri/TicketView/issues/19#issue-499865059

### CODE REVIEW

**Comment by vipulasri** [comment]
@shishanksingh2015 What are you trying to achieve here? Can you please elaborate and would be helpful if you can also share the screenshots of before and after.

**Comment by shishanksingh2015** [comment]
updated the PR detail.please review.

File: README.md [hunk 1]
```diff
@@ -166,6 +166,16 @@ If you Watch this repository, GitHub will send you an email every time I publish
         <td>0dp</td>
         <td>sets elevation to ticket view on android jellybean and above</td>
     </tr>
+    <tr>
+        <td>app:ticketBackgroundBelowDivider</td>
+         <td>none</td>
+         <td>sets background to ticket view below divider on android jellybean and above</td>
```
Comment (vipulasri) [inline README.md:172]: The statement over here is slightly wrong, this drawable can be applied on any android version.

File: README.md [hunk 2]
```diff
@@ -166,6 +166,16 @@ If you Watch this repository, GitHub will send you an email every time I publish
...
+        </tr>
+        <tr>
+          <td>app:ticketBackgroundAboveDivider</td>
+          <td>none</td>
+          <td>sets background to ticket view above divider on android jellybean and above</td>
```
Comment (vipulasri) [inline README.md:177]: same here.

File: README.md (hunk 1 above)
Comment (shishanksingh2015) [reply README.md]: fixed in both

**Pull request merged**